fix and improve content on reference pages of table elements (#27958)
mdn/content: files/en-us/web/html/element/caption/index.md

- `align` {{deprecated_inline}}

>   - : This {{Glossary("enumerated", "enumerated")}} attribute indicates how the caption must be aligned with respect to the table. Possible values are:

Author: I like `accepts` the most, because the attribute _still_ allows the values
```suggestion
  - : This deprecated {{Glossary("enumerated", "enumerated")}} attribute accepts `left`, `top`, `right`, or `bottom` as a value to indicate which side of the table the caption should be displayed. Use the {{cssxref("caption-side")}} and {{cssxref("text-align")}} CSS properties instead.
```

>   - : This {{Glossary("enumerated", "enumerated")}} attribute indicates how the caption must be aligned with respect to the table. Possible values are:

Author: After giving it some thought, I have come up with the following sentence structure, which I think is clear and at the same time short and concise:
> This {{Glossary("enumerated", "enumerated")}} attribute defines which side of the table the caption should be displayed. Possible values are `left`, `top`, `right`, or `bottom`. Use the {{cssxref("caption-side")}} and {{cssxref("text-align")}} CSS properties instead, as this attribute is deprecated.

I'll go with this structure for the other deprecated attributes too. I hope that this is fine for you.

    - `left`: the caption is displayed to the left of the table;
    - `top`: the caption is displayed above the table;
    - `right`: the caption is displayed to the right of the table;

>     - `bottom`: the caption is displayed below the table.

Author: Yes, I think the notes are repetitive (mentioning the deprecation) and are too present / take up too much vertical space on the page... Especially when there are a lot of deprecated attributes.

    > **Note:** Do not use this attribute as it is deprecated (and not supported) in the latest standard: use the CSS {{cssxref("caption-side")}} and {{cssxref("text-align")}} properties instead.

## Usage notes

- If used, the `<caption>` element must be the first child of its parent {{htmlelement("table")}} element.

> - When the {{HTMLElement("table")}} element that contains the `<caption>` is the only descendant of a {{HTMLElement("figure")}} element, you should use the {{HTMLElement("figcaption")}} element instead of `<caption>`.

Author: I'll go with this small tweak, I hope your fine with it:
```suggestion
- When a `<table>` is nested within a {{HTMLElement("figure")}} as the figure's only content, it should be captioned via a {{HTMLElement("figcaption")}} for the `<figure>` instead of as a `<caption>` nested within the `<table>`.
```

## Example

See {{HTMLElement("table")}} for a complete table example introducing common standards and best practices.

This example shows a basic table that includes a caption describing the data presented. Some basic CSS is used to align and highlight the `<caption>`.

Such a "title" is helpful for users who are quickly scanning the page, and it is especially beneficial for visually impaired users, allowing them to determine the table's relevance quickly without the need to have a screen reader read the contents of many cells just to find out what the table is about.

```html
<table>
  <caption>
    User login email addresses
  </caption>
  <tr>
    <th>Login</th>
    <th>Email</th>
  </tr>
  <tr>
    <td>user1</td>
    <td>user1@example.com</td>
  </tr>
  <tr>
    <td>user2</td>
    <td>user2@example.com</td>
  </tr>
</table>
```

```css
caption {
  caption-side: top;
  text-align: left;
  padding-bottom: 10px;
  font-weight: bold;
}
```

```css hidden
table {
  border-collapse: collapse;
  border: 2px solid rgb(140, 140, 140);
  font-family: sans-serif;
  font-size: 0.8rem;
  letter-spacing: 1px;
}

th,
td {
  border: 1px solid rgb(160, 160, 160);
  padding: 8px 10px;
}

th {

>   background-color: rgb(230, 230, 230);

Author: You're right... Out of sheer outdated habit...
